projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -314,6 +334,9 @@ export function getPlanClass( plan ) {
 		case PLAN_JETPACK_BACKUP_REALTIME:
 		case PLAN_JETPACK_BACKUP_REALTIME_MONTHLY:
 			return 'is-realtime-backup-plan';
+		case PLAN_JETPACK_VIDEOPRESS:
+		case PLAN_JETPACK_VIDEOPRESS_MONTHLY:
+			return 'is-videopress-plan';
```

Comment:
These appear nested under the "DEPRECATED" comment on L323, is that correct?

Reply:
Not correct! Fixed in e23cab2.